impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -226,6 +217,10 @@ private KafkaContainer buildKafkaContainer(KafkaClusterConfig.ConfigHolder holde
             kafkaContainer.addFixedExposedPort(holder.getExternalPort(), CLIENT_PORT);
             kafkaContainer.addFixedExposedPort(holder.getAnonPort(), ANON_PORT);
         }
+        else {
+            // disables automatic configuration of broker role and default listeners by kafka-native
+            kafkaContainer.withEnv("SERVER_AUTO_CONFIGURE", "false");
```

Comment:
what's stopping us setting this flag in the `holder.isBroker() == true` case?  what are we missing from the configuration?

Reply:
I see you did this. resolving comment.